maraboupy/MarabouCore.cpp

```diff
@@ -376,42 +376,6 @@ struct MarabouOptions
 };
 
 
-/* The default parameters here are just for readability, you should specify
```

Comment:
I believe this method is not needed given that we already have calculuateBounds(). Writing a deep-copy constructor for the context dependent InputQuery class just for this seems a bit unncessary.